cli/types/cypress.d.ts

```diff
@@ -2609,6 +2609,109 @@ declare namespace Cypress {
     includeShadowDom: boolean
   }
 
+  /**
+   * Options appended to config object on runtime.
+   */
+  interface RuntimeConfigOptions {
+    /**
+     * CPU architecture, from Node `os.arch()`
+     *
+     * @see https://nodejs.org/api/os.html#os_os_arch
+     */
+    arch: string
+    /**
+     * The list of hosts to be blocked
+     */
+    blockHosts: null | string | string[]
+    /**
+     * The browser Cypress is runnong on.
+     */
+    browser: Browser
+    /**
+     * The browsers Cypress can use.
+     */
+    browsers: Browser[]
+    /**
+     * Path to folder containing component test files.
+     */
+    componentFolder: string
+    /**
+     * Enables component testing.
+     */
+    experimentalComponentTesting: boolean
+    /**
+     * Enables polyfill fetch.
+     */
+    experimentalFetchPolyfill: boolean
+    /**
+     * ???
+     */
+    hosts: null | string[]
+    /**
+     * With this option enabled - Cypress will search through the response streams
+     * coming from your server on .html and .js files
+     * and replace code that matches some patterns.
+     *
+     * @see https://docs.cypress.io/guides/references/configuration.html#modifyObstructiveCode
+     */
+    modifyObstructiveCode: boolean
+    /**
+     * ???
+     */
+    projectId: null | string
+    /**
+     * The path to the support folder
+     */
+    supportFolder: string
+    /**
+     * Glob pattern for the test files.
+     */
+    testFiles: string
+    /**
+     * ???
+     */
+    userAgent: null | string
+    /**
+     * The Cypress version being used.
+     */
+    version: string
+
+    // Internal or Unlisted at server/lib/config_options
+    autoOpen: boolean
+    browserUrl: string
+    clientRoute: string
+    configFile: string
+    cypressEnv: string
+    integrationExampleName: string
+    integrationExamplePath: string
+    isInteractive: boolean
```

Comment:
This one is ok to document, we use it in a lot of examples.
```suggestion
    /**
     * Whether Cypress was launched via 'cypress open' (interactive mode)
     */
    isInteractive: boolean
```